client/src/pages/Login.js

```diff
@@ -0,0 +1,111 @@
+import React, { useState } from "react";
+
+import {
+  Button,
+  TextField,
+  Link,
+  Typography,
+  Container,
+} from "@material-ui/core";
+
+import { makeStyles } from "@material-ui/core/styles";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    marginTop: theme.spacing(8),
+    display: "flex",
+    flexDirection: "column",
+    alignItems: "center",
+    borderRadius: "10px",
+    boxShadow: "0 0 5px 0",
+    padding: theme.spacing(3, 5),
+    backgroundColor: "white",
+  },
+  underline: {
+    width: "50px",
+    height: "4px",
+    color: "#32BE72",
+    backgroundColor: "#32BE72",
+  },
+  form: {
+    marginTop: theme.spacing(1),
+    width: "100%",
+  },
+  formButton: {
+    display: "flex",
+    margin: "24px auto 32px",
+    textTransform: "none",
+    backgroundColor: "#32BE72",
+    color: "white",
+    width: "120px",
+    height: "40px",
+  },
+}));
+
+export default function Login() {
```

Comment:
I saw in some files, you use export default at the bottom of the file. Would you please make it consistent.